README.md

```diff
@@ -8,10 +8,10 @@ For a quick installation:
 
 1. Install dependencies: python 2.7, `pip`, [`virtualenv`](https://virtualenv.pypa.io/en/latest/installation/), git, [git-lfs](https://git-lfs.github.com/).
 
-2. Clone this repository: 
+2. Clone this repository:
 
     ```bash
-    git clone git@github.com:Exabyte-io/documentation.git
+    git clone https://github.com/Exabyte-io/documentation.git
```

Comment:
@pranabdas Glad to hear from you here - thank you for the input!

Let's add both options.

Setting up an SSH authentication once with github allows one to avoid having to authenticate with password(s) **ever** after. HTTPS requires additional authentication on each action and when dealing with submodules for each of them, if I remember correctly. But I do see why this could be helpful to new contributors.

Additionally, using SSH is convenient inside the workflow pipeline where the HTML sources are built.

How about explaining how `.gitmodules` can be adjusted to use HTTPS for novice users in the README and keeping SSH in `.gitmodules` as a way forward?

Reply:
@timurbazhirov why do I even need to authenticate in order to clone a public repository (which is the case here)? 🤔 You can still keep private repo URL protocols SSH.

Reply:
OK, good point! Submodule was added before the codebase was made public indeed. It looks like everything should work with HTTPS.

```bash
 mat3ra-documentation git:(git-https) git submodule update --init     
Submodule 'esse' (https://github.com/Exabyte-io/exabyte-esse.git) registered for path 'data/esse'
```